---
title: "The *GRaNIEData* package"
author: "Christian Arnold"
date: "`r doc_date()`"
package: "`r BiocStyle::pkg_ver('GRaNIEData')`"
abstract: >
  This is a companion package for *GRaNIE* and provides additional example datasets for it. The datasets in the *GRaNIEData* package are used in the *GRaNIE* vignette and provide a real-world dataset suitable for analyses.

vignette: >
  %\VignetteIndexEntry{The *GRaNIEData* package}
  %\VignetteEngine{knitr::rmarkdown}
  %\VignetteEncoding{UTF-8}
output: 
  BiocStyle::html_document
---

# Description of the dataset and the aim of the analysis

The data is taken from [here](https://zenodo.org/record/1188300#.X370PXUzaSN) [1], has been processed to meet the requirements of the `GRaNIE` package and consists of the following files:

* ATAC-seq peaks (around 75.000, genome-wide)
* RNA-Seq data for around 35.000 genes (raw counts)
* sample metadata with additional sample-specific information

In general, the dataset is from human macrophages (both naive and IFNg primed) of healthy individuals and various stimulations / infections (naive vs primed and infected with SL1344 vs not), with 4 groups in total: control/infected(SL1344) and naive/primed(IFNg)


Furthermore, the example dataset is accompanied by the following files:

* genome-wide transcription factor binding site predictions for 75 selected TFs, along with a translation table to link TF names to their corresponding Ensembl IDs. For each TF, a gzipped BED file has been created with predicted TF binding sites. The files have been generated with `PWMScan` and the `HOCOMOCO` database, see [2] for details.


# Example Workflow

For an example workflow, see the workflow vignette of the `GRaNIE` package.


# References
[1] https://zenodo.org/record/1188300#.X370PXUzaSN
[2] Ivan Berest*, Christian Arnold*, Armando Reyes-Palomares, Giovanni Palla, Kasper Dindler Rasmussen, Holly Giles, Peter-Martin Bruch, Wolfgang Huber, Sascha Dietrich, Kristian Helin, Judith B. Zaugg. Quantification of Differential Transcription Factor Activity and Multiomics-Based Classification into Activators and Repressors: diffTF. 2019. Cell Reports 29(10), P3147-3159.E12.

# Session Info

```{r}
 sessionInfo()
